charts/trino/templates/configmap-worker.yaml

```diff
@@ -33,17 +33,17 @@ data:
     -XX:PerMethodRecompilationCutoff=10000
     -XX:PerBytecodeRecompilationCutoff=10000
     -Djdk.nio.maxCachedBufferSize=2000000
-  {{- range $configValue := .Values.additionalJVMConfig }}
+  {{- range $configValue := .Values.worker.additionalJVMConfig }}
     {{ $configValue }}
   {{- end }}
 
   config.properties: |
     coordinator=false
     http-server.http.port={{ .Values.service.port }}
-    query.max-memory={{ .Values.server.config.query.maxMemory }}
-    query.max-memory-per-node={{ .Values.server.config.query.maxMemoryPerNode }}
-    query.max-total-memory-per-node={{ .Values.server.config.query.maxTotalMemoryPerNode }}
-    memory.heap-headroom-per-node={{ .Values.server.config.memory.heapHeadroomPerNode }}
+    query.max-memory={{ .Values.worker.config.query.maxMemory }}
+    query.max-memory-per-node={{ .Values.worker.config.query.maxMemoryPerNode }}
+    query.max-total-memory-per-node={{ .Values.worker.config.query.maxTotalMemoryPerNode }}
+    memory.heap-headroom-per-node={{ .Values.worker.config.memory.heapHeadroomPerNode }}
```

Comment:
Worker level values don't matter for things like `query.max-memory`. This change will mislead chart users and leave them with a system that looks like it doesn't honor the configs.